TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceUI/Scripts/Configuration/ConfigUI/InteractionSettingsConfigUI.cs

```diff
@@ -209,6 +210,35 @@ protected override void LoadConfigValuesIntoFields()
             DisplayIntractionPreview();
         }
 
+        void OnSliderValueChange(float _)
+        {
+            if (sliderValueChangedDelay <= 0)
+            {
+                sliderValueChangedDelay = 0.1f;
+                StartCoroutine(SetValueChangedAfterDelay());
+            }
+            else
+            {
+                sliderValueChangedDelay = 0.1f;
+            }
+        }
+
+        float sliderValueChangedDelay = 0;
```

Comment:
For tidiness. should this be declared at the top of the file rather than in the middle ?